src/Cloneable.php

```diff
@@ -70,21 +70,22 @@ public function addCloneableRelation($relation) {
 
 	/**
 	 * Clone the current model instance
-	 *
+	 * @param  array $attr Extra attributes for each clone
 	 * @return \Illuminate\Database\Eloquent\Model The new, saved clone
 	 */
-	public function duplicate() {
-		return App::make('cloner')->duplicate($this);
+	public function duplicate($attr = null) {
+		return App::make('cloner')->duplicate($this, null, $attr);
 	}
 
 	/**
 	 * Clone the current model instance to a specific Laravel database connection
 	 *
 	 * @param  string $connection A Laravel database connection
+	 * @param  array $attr Extra attributes for each clone
 	 * @return \Illuminate\Database\Eloquent\Model The new, saved clone
 	 */
-	public function duplicateTo($connection) {
-		return App::make('cloner')->duplicateTo($this, $connection);
+	public function duplicateTo($connection, $attr) {
```

Comment:
Should all the new `$attr` arguments have default values?  Like:

```suggestion
	public function duplicateTo($connection, $attr = null) {
```

Reply:
Oh, yeah, forgot to add that to the PR, did add it to my tests